plugin/keyring_vault/vault_parser.cc

```diff
@@ -0,0 +1,178 @@
+#include "vault_parser.h"
```

Comment:
I can only imagine how much problem it would be to add another dependency. It is simple parsing here, thus vault_parser should be enough.